saleor/payment/gateways/braintree/__init__.py

```diff
@@ -33,23 +31,31 @@
 }
 
 
-def get_customer_data(payment: Payment) -> Dict:
+class TransactionKind:
```

Comment:
Couldn't we have one `TransactionKind` definition? I see that we have multiple ones and they are identical.

Reply:
See https://github.com/mirumee/saleor/pull/3519#discussion_r247162323 and https://github.com/mirumee/saleor/pull/3519#discussion_r247901262 (re common types).